common/templates/includes/map.njk

```diff
@@ -0,0 +1,219 @@
+<div class="govuk-grid-column-full" style="height: 600px; position: relative">
+  <div id="map" class="map" class="govuk-grid-column-full" style="height: 600px"></div>
+  <div id="message" class="govuk-grid-column-full govuk-heading-l" style="position: absolute; top: 100px; height: 200px; text-align: center"></div>
+</div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol.js" integrity="sha512-O7kHS9ooekX8EveiC94z9xSvD/4xt10Qigl6uEKvspYykdux3Ci5QNu5fwi4ca0ZkZI/oCgx5ja8RklWUEqzxQ==" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol.css" integrity="sha512-D2c/nmSVlwoZk29SLwJK8GkLyCyPuNbZ/vZQL/lNzw+o+yXzsbnJt7lK/iCfdBmgyLRDi0qx27gpE8Zb1C6I7g==" crossorigin="anonymous" />
+{#<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/openlayers/4.6.5/ol-debug.css" integrity="sha512-hBSieZLd5rse9gdkfv4n0pDU4D04SxpqBtwDzRy/QiXRBhczDyfCTDTnHCada73ubNqiQv6BLgCRXHAJPUwC5w==" crossorigin="anonymous" />#}
```

Comment:
Rogue comments?

Reply:
These are not rogue comments - as mentioned in https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1373#discussion_r587460110 I've left the debug URLs commented out for now, as they are not obvious to determine and will probably be useful in the future.

If we decide we want some more dynamic way of using them, we could build something off a query string in the future.